doc/adr/0003-publisher-not-using-react.md

```diff
@@ -7,11 +7,18 @@ Date: 2019-10-01
 Accepted
 
 ## Context
-Libero Publisher needs a read-optimised, highly performant, progressively enhanced front end.
+Investigation into stronger alignment of technologies, practices and processes across the Libero Suite of products has resulted in time-boxed discovery sessions to find the advantages and consequences of various different aspects.
+One such area was front-end technologies. Other areas looked at aligning around JavaScript technologies and removing PHP from Libero Publisher.
```

Comment:
Not pertinent to this ADR: "Other areas looked at aligning around JavaScript technologies and removing PHP from Libero Publisher."?

Reply:
I thought it was pertinent since it gives the context as to why we're considering a JavaScript based technology so strongly.

Reply:
Okay, yeah that makes sense.